docs/src/variables.md

```diff
@@ -42,7 +42,7 @@ This code does three things:
 2. it creates two *JuMP* variables that act as references to those optimization
    variables
 3. it binds those JuMP variables as a vector with two elements to the *Julia*
```

Comment:
The formatting of this list is inconsistent with the one below. The suggestion at https://developers.google.com/style/lists is to use complete sentences with proper capitalization for all list items. Could you make this change here also?

Reply:
Done!